### ISSUE TITLE
TensorFlow is not turning on for the backend after running this cell with Python code. Check for any errors in the lines. 

### ISSUE THREAD

**Issue by AniFineArt** [AniFineArt/DIP_Py_1#1]
![screenshot-colab research google com-2020 12 09-10_07_11](https://user-images.githubusercontent.com/59864902/101585719-5bd78d80-3a06-11eb-8262-ae80610bb36d.png)

#Import the libraries
import tensorflow as tf 
from tensorflow import keras
from keras.models import Sequential
from keras.layers import Dense, Flatten, Conv2D, MaxPooling2D, Dropout
from tensorflow.keras import layers
from keras.utils import to_categorical
import numpy as np
import matplotlib.pyplot as plt
plt.style.use ('fivethirtyeight')
